queue_services/entity-filer/src/entity_filer/filing_processors/consent_continuation_out.py

```diff
@@ -0,0 +1,38 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the court order filing."""
+from contextlib import suppress
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from typing import Dict
+
+from legal_api.models import Business, Document, DocumentType, Filing
+
+from entity_filer.filing_meta import FilingMeta
+
+
+def process(business: Business, cco_filing: Filing, filing: Dict, filing_meta: FilingMeta):
+    """Render the court order filing into the business model objects."""
+    cco_filing.court_order_file_number = filing['courtOrder'].get('fileNumber')
+    cco_filing.court_order_effect_of_order = filing['courtOrder'].get('effectOfOrder')
+    cco_filing.order_details = filing['orderDetails']
+
+    business.cco_expiry_date = datetime.now() + relativedelta(months=6)
```

Comment:
Was "cco_expiry_date" changed to "expiry"? I thought I saw @argush3 say something about this.

Reply:
yes, it's been changed to `expiry`.  Good catch Chris.    

I put a note regarding this in the ticket.  The FE was already using `expiry`.

Reply:
nevermind.  this is right.  `filing.meta_data` is what needs to be populated with `expiry` which is done below.